plugins/com.google.cloud.tools.eclipse.appengine.newproject.maven/src/com/google/cloud/tools/eclipse/appengine/newproject/maven/CreateMavenBasedAppEngineStandardProject.java

```diff
@@ -57,7 +57,7 @@ protected void execute(IProgressMonitor monitor)
       appId = artifactId;
     }
     String appengineArtifactVersion = resolveLatestReleasedArtifact(progress.newChild(20),
-        "com.google.appengine", "appengine-api-1.0-sdk", "jar", "1.9.37");
+        "com.google.appengine", "appengine-api-1.0-sdk", "jar", "1.9.42");
```

Comment:
This method actually tries to resolve the latest and uses the provided version as a fall-back.  So we are putting out 1.9.42 anyways.